### PR TITLE
Le Train Dispatcher

### PR DESCRIPTION
**Pull request by Athorus**
News about a new game : Le Train Dispatcher.

### CODE REVIEW

File: _drafts/newsletter-004.md [hunk 1]
```diff
@@ -44,6 +44,45 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+![Demo of Le Train Dispatcher](/assets/newsletter-004/ltd.gif)
+
+Le Train Dispatcher allows you to route trains in a fully simulated rail network. Particular care has been taken on the realistic management of light signals (block systems, switches protection), train physics and curve tracing.
+
+The game is programmed in Rust and the main crates used are : ggez, imgui and serde.
+
+This first version is fully playable, but if you want to comment, bring your ideas or help the development of the game, in any way, do not hesitate to post a [message][ltd-post].
+
+#### Unique characteristics
```
Comment (AlexEne) [inline _drafts/newsletter-004.md]: This can be combined with the part above

**Comment by ozkriff** [comment]
You don't need to close this PR. Just push more commits to your branch: https://stackoverflow.com/questions/16748115/how-to-modify-github-pull-request

File: _drafts/newsletter-004.md [hunk 2]
```diff
@@ -44,6 +44,45 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+    - Huge engineering work has been done to have a realistic management of railway signaling. 
+- No collision
+    - Yes it's fun to watch  many trains running automatically without getting in. All your actions are checked : you cannot put the trains in danger.
+- True physics
+    - Each locomotive or wagon has its own weight, its driving or braking force, its coefficient of adhesion.
+    - The curves are not simple circles, they are calculated with the same equations as those used in real road or rail networks.
+
+
+#### Help needed!
+Creating a game it is a team effort, I am looking for partners first of foremost. If you have one of the talents below, you are welcome.
```
[the remark this answers is not in the document]
Comment (ozkriff) [reply _drafts/newsletter-004.md]: I guess a shorter version of this is fine for the "Requests for Contribution" section?

File: _drafts/newsletter-004.md [hunk 3]
```diff
@@ -44,6 +44,45 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+- Graphic artist
+- Level and game designer
+- Sound engineer
+- Web designer
+- Musician
+
+#### Links
```
Comment (ozkriff) [inline _drafts/newsletter-004.md]:
```suggestion
Links: [itch.io][ltd], [Patreon][ltd-patreon]
```

I think it's better to use a one-liner for this.

**Comment by Athorus** [comment]
> Thanks for the PR!
> 
> You don't need to close this PR to edit it - just push more commits to your branch: https://stackoverflow.com/questions/16748115/how-to-modify-github-pull-request

**Comment by Athorus** [comment]
Sorry for closing this PR, I was confused by the interface.

**Comment by Athorus** [comment]
Here my changes. Sorry about the number of commits.

**Comment by 17cupsofcoffee** [comment]
@Athorus: Don't worry about it, there's no good way to go back and amend commits through the web interface. They can be squashed down when the change merges 🙂

**Comment by Athorus** [comment]
Thanks. :) Do i have something else to do ?

**Comment by ozkriff** [comment]
Merged!

**Pull request merged**